client/landing/stepper/declarative-flow/sensei.ts

```diff
@@ -0,0 +1,101 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { SENSEI_FLOW, useFlowProgress } from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { ONBOARD_STORE, USER_STORE } from '../stores';
+import Launchpad from './internals/steps-repository/launchpad';
+import ProcessingStep from './internals/steps-repository/processing-step';
+import SenseiDomain from './internals/steps-repository/sensei-domain';
+import SenseiLaunch from './internals/steps-repository/sensei-launch';
+import SenseiPlan from './internals/steps-repository/sensei-plan';
+import SenseiSetup from './internals/steps-repository/sensei-setup';
+import { AssertConditionState, Flow } from './internals/types';
+import './sensei.scss';
+
+const sensei: Flow = {
+	name: SENSEI_FLOW,
+	title: 'Sensei',
+	useSteps() {
+		useEffect( () => {
+			recordTracksEvent( 'calypso_signup_start', { flow: this.name } );
+			recordFullStoryEvent( 'calypso_signup_start_sensei', { flow: this.name } );
+		}, [] );
+
+		return [
+			{ slug: 'senseiSetup', component: SenseiSetup },
+			{ slug: 'senseiDomain', component: SenseiDomain },
+			{ slug: 'senseiPlan', component: SenseiPlan },
+			{ slug: 'senseiLaunch', component: SenseiLaunch },
+			{ slug: 'launchpad', component: Launchpad },
+			{ slug: 'processing', component: ProcessingStep },
+		];
+	},
+
+	useStepNavigation( _currentStep, navigate ) {
+		const flowName = this.name;
+		const { setStepProgress } = useDispatch( ONBOARD_STORE );
+		const flowProgress = useFlowProgress( { stepName: _currentStep, flowName } );
+		const siteSlug = useSiteSlug();
+		setStepProgress( flowProgress );
+
+		const goBack = () => {
+			return;
+		};
+
+		const goNext = () => {
+			switch ( _currentStep ) {
+				case 'senseiSetup':
+					return navigate( 'senseiDomain' );
+				case 'senseiDomain':
+					return navigate( 'senseiPlan' );
+				case 'launchpad':
+					return window.location.assign( `/view/${ siteSlug }` );
+				default:
+					return navigate( 'launchpad' );
+			}
+		};
+
+		const submit = () => {
+			switch ( _currentStep ) {
+				case 'senseiSetup':
+					return navigate( 'senseiDomain' );
+				case 'senseiDomain':
+					return navigate( 'senseiPlan' );
+				case 'launchpad':
+					return navigate( 'processing' );
+				default:
+					return navigate( 'launchpad' );
+			}
+		};
+
+		const goToStep = ( step: string ) => {
+			navigate( step );
+		};
+
+		return { goNext, goBack, goToStep, submit };
```

Comment:
```suggestion
		return { submit };
```

Reply:
Fixed in f2da75ffa9b20b20abf3b1f4b8c14da042610ddb